.github/workflows/compliance.yml

```diff
@@ -41,9 +41,16 @@ jobs:
         #[[ "$(git rev-list --merges --count origin/${BASE_REF}..)" == "0" ]] || \
         #(echo "::error ::Merge commits not allowed, rebase instead";false)
         # Sauce tag checks before rebasing
-        git rev-list --first-parent origin/${BASE_REF}..HEAD | paste -sd, | \
-        xargs gitlint -c ncs-sauce-tags.enable=true \
-        -c title-starts-with-subsystem.regex=".*" --commits
+        # Handle single commit separately
+        if [[ "$(git rev-list --first-parent --count origin/${BASE_REF}..)" == "1" ]]; then
```

Comment:
```
if [[ "$(git rev-list --first-parent --count origin/${BASE_REF}..)" == "1" ]]; then COMMITS="--commit"; else COMMITS="--commits"; fi
git rev-list --first-parent origin/${BASE_REF}..HEAD | paste -sd, | \
xargs gitlint -c ncs-sauce-tags.enable=true \
-c title-starts-with-subsystem.regex=".*" $COMMITS
```

Reply:
Thanks, this is what I did earlier, but to avoid breaking moved to if..else.